airflow/models/baseoperator.py

```diff
@@ -1874,6 +1794,34 @@ def get_serialized_fields(cls):
             )
         return cls.__serialized_fields
 
+    @property
+    def params(self) -> Union[dict, ParamsDict]:
+        return self.partial_kwargs.get("params", ParamsDict())
+
+    @property
+    def queue(self) -> str:
+        return self.partial_kwargs.get("queue", DEFAULT_QUEUE)
```

Comment:
Hmmm, this means that queue property can't be mapped, right?

Reply:
We need _some_ value on the unmapped TI since `queue` (and other calculated properties listed here) is recorded when the (unmapped) TI is created. We can still implement mapping support by using a different value when we create the mapped TIs, but we still need this so the unmapped TI has something to store in the db.

Reply:
Fair point.